test/functional/actions/sendEvent.spec.js

```diff
@@ -156,6 +153,7 @@ test("returns full valid settings with decision scopes as data element", async (
   });
   await instanceNameField.selectOption("alloy2");
   await typeField.enterSearch("mytype1");
+  await typeField.pressEnterKey();
```

Comment:
I upgraded React-Spectrum's ComboBox and their new version requires hitting enter to accept the text instead of just blurring.